doc/source/addons/RemoteTech.rst

```diff
@@ -29,6 +32,16 @@ It is possible to activate/deactivate RT antennas, as well as set their targets
 
 Acceptable values for `"target"` are: `"no-target"`, `"active-vessel"`, `"mission-control"`, a :struct:`Body`, a :struct:`Vessel`, or a string containing the name of a body or vessel.
 
+Communication
+~~~~~~~~~~~~~
+
+RemoteTech will influence :ref:`communication <communication>` between vessels. In order to send a message to another vessel a valid RemoteTech connection will have to exist between them
```

Comment:
I'd add the qualifier that you mean *if it's installed* then a RemoteTech connection is required.  It's my understanding that you can still do inter-vessel comms without RemoteTech, but you just don't get delays and connection checks then.  (Is this right?).